### PR TITLE
Showing how to access specific data items

### PR DESCRIPTION
**Pull request by davidsilvasmith**
This patch is an update to the docs showing how to access specific data items via front matter variables.

### CODE REVIEW

**Comment by parkr** [comment]
This has your LSI docs update as well :(

File: site/_docs/datafiles.md [hunk 1]
```diff
@@ -114,3 +114,33 @@ The organizations can then be accessed via `site.data.orgs`, followed by the fil
...
+The author can then be specified as a page variable in a post's frontmatter:
+
+{% highlight html %}
+{% raw %}
+---
+title: sample post
+author: dave
+---
+
+{% assign author = site.data.people.[page.author] %}
```
Comment (parkr) [inline site/_docs/datafiles.md]: Isn't the `.` after `people` invalid?

**Comment by parkr** [comment]
One comment, then I'll merge! Thanks, Dave!

File: site/_docs/datafiles.md (hunk 1 above)
Comment (davidsilvasmith) [inline site/_docs/datafiles.md]: It's a weird syntax (super useful though, took me some googling to get it). I confirmed it's working correctly on my site: https://github.com/davidsilvasmith/bitcoinbulls.net/blob/gh-pages/_layouts/post.html

Sorry about including the LSI docs. I'm new to pull requests. I'll be better with future pulls :)

Comment (parkr) [inline site/_docs/datafiles.md]: Does it work without the added dot? It will likely break in Liquid 3's stricter lexer.

**Pull request merged**